frontend/src/services/orderService.js

```
@@ -0,0 +1,21 @@
+import axios from 'axios';
+
+const API_URL = import.meta.env.VITE_API_URL || 'http://localhost:5000/api';
+
+export const placeOrder = async (orderData) => {
+  try {
+    const response = await axios.post(`${API_URL}/orders`, orderData);
+    return response.data;
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)